nirc_ehr/src/org/labkey/nirc_ehr/NIRC_EHRModule.java

```diff
@@ -79,7 +80,7 @@ public String getName()
     @Override
     public @Nullable Double getSchemaVersion()
     {
-        return 24.006;
+        return 24.006; //TODO
```

Comment:
TODO for missing upgrade?

Reply:
Good catch, I had this as a TODO in case you got your changes with the same schema version merged before mine. Now updated.